website/src/pages/case-studies/lyft-monolith-to-microservices.tsx

```diff
@@ -58,30 +49,75 @@ hundreds of repositories, so a high level of accuracy is needed."
                     />
 
                     <p>
-                        Read on to learn how Lyft used Sourcegraph code search to provide a company wide verification
-                        stage to confirm all call sites to previous monolith code had been updated before deploying to
+                        Learn how Lyft used Sourcegraph code search to provide a company wide verification stage to
+                        confirm all call sites to previous monolith code had been updated before deploying to
                         production.
                     </p>
 
-                    <h2 className="pt-5 pb-1">HEADING</h2>
-                    <p>content</p>
+                    <h2 className="pt-5 pb-1">From monolith to microservices</h2>
+
+                    <p>
+                        The largest refactoring effort in Lyft's 10 year history came in 2018 when the number one
+                        company priority was the decomposition of the PHP monolith to microservices.
+                    </p>
+
+                    <p>
+                        Performing such significant and wide-ranging code changes meant extensive analysis was required
+                        to verify all call sites and references to the monolith has been updated. This meant
+                        transforming monolioth code to consuming the new microservice API endpoints, which in many
+                        cases, meant altering the API schema.
+                    </p>
+
+                    <InContentBlockquote
+                        quote="Sourcegraph code search helped ensure production stability throughout the monolith to microservices decomposition by verifying applications were no longer pointing at the monolith version of a service."
+                        author="Justin Phillips, Software Engineer"
+                    />
+
+                    {/*This case study demonstrates how Lyft used Sourcegraph code search to aid in the decomposition of their 10 year old PHP monolith to microservices.*/}
+
+                    <p>
+                        This wasn't simply slicing up the monolith by domain into separate services—it was also a
```

Comment:
Add blank on either side of the dash between `services` and `it`.

Reply:
It's an em dash so typically, there is no space either side.